src/applications/simple-forms/21-0845/pages/authAddr.js

```diff
@@ -0,0 +1,16 @@
+import definitions from 'vets-json-schema/dist/definitions.json';
+import { uiSchema, schema } from '../definitions/form-address';
+
+/** @type {PageSchema} */
+export default {
+  uiSchema: {
+    authorizerAddress: uiSchema({ root: '' }),
+  },
+  schema: {
+    type: 'object',
+    required: ['authorizerAddress'],
+    properties: {
+      authorizerAddress: schema({ definitions }, true),
```

Comment:
hm, do we want to pass in all the definitions here? maybe its a non-issue to have more definitions than you need

Reply:
Yes, I think the reason why our address def wants all the definitions from definitions.json passed in is, it can set a default to select "address" from the definitions, and let devs override it if they need "usAddress" or "profileAddress.